components/new_channel_flow/index.ts

```diff
@@ -28,7 +21,7 @@ function mapStateToProps(state: GlobalState) {
 
 function mapDispatchToProps(dispatch: Dispatch<GenericAction>) {
     return {
-        actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc>, Actions>({
+        actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc>, Props['actions']>({
```

Comment:
That's a nice way to avoid duplicating this. Ideally, I don't think we'd want to specify the types here at all, but this is still a lot nicer than before.